controller/workitem.go

```diff
@@ -157,7 +161,11 @@ func (c *WorkitemController) Show(ctx *app.ShowWorkitemContext) error {
 	return ctx.ConditionalRequest(*wi, c.config.GetCacheControlWorkItem, func() error {
 		comments := workItemIncludeCommentsAndTotal(ctx, c.db, ctx.WiID)
 		hasChildren := workItemIncludeHasChildren(ctx, c.db)
-		wi2 := ConvertWorkItem(ctx.Request, *wi, comments, hasChildren)
+		wit, err := c.db.WorkItemTypes().Load(ctx.Context, wi.Type)
```

Comment:
Looks like this call and the above call to get children are under `ctx.ConditionalRequest`. I think these calls can be moved under the above transaction wrapper.

Reply:
Done in bb6a73d8